content/katas/fibonacci-sequence/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Fibonacci sequence
+pre: "<b>1. </b>"
+weight: 5
+---
+
+
+ **What is fibonacci sequence?**
+
+> Fibonacci Sequence: is the series of numbers where the next number is found by adding up the 2 numbers before it.
+> in this case n = 2:
+> sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, ...
+
+**1.** 
+Write a function, fibonacci(n) that takes in a number/int argument (n) and returns the fibonacci sequence from the 1st term of the sequence till the nth term of the sequence. 
+
+```
+
+function fibonacci(n)
+```
+```
+fibonacci(144)
```

Comment:
This is incorrect because `fibonacci(144)` is not `[0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]`

but `fibonacci(12)` is `[0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89]`
because this returns the first 12 terms of the sequence.